ple/games/vgdl/core.py

```diff
@@ -0,0 +1,743 @@
+'''
+Video game description language -- parser, framework and core game classes.
+
+@author: Tom Schaul

```

Comment:
Looks like a BSD 3 clause. Found here https://github.com/schaul/py-vgdl/blob/master/LICENSE.txt . I'll add these to the relevant files.